File: 1 - Swift/Including Swift Files in Objective C Project/ObjectiveCApp/ViewController.m
//
//  ViewController.m
//  ObjectiveCApp
//
//  Created by Nick Kuh on 17/04/2015.
//  Copyright (c) 2015 Mumbo Apps Ltd. All rights reserved.
//

#import "ViewController.h"
#import "ObjectiveCApp-Swift.h"

@implementation ViewController

- (IBAction)didTapButton:(id)sender {
    MySwiftViewController *vc = [[MySwiftViewController alloc] initWithNibName:@"MySwiftViewController" bundle:nil];
    [self.navigationController showViewController:vc sender:self];
}

@end

File: My Swift Project with Cocoapods/After With Bridging Header/My Swift Project with Cocoapods/ViewController.swift
//
//  ViewController.swift
//  My Swift Project with Cocoapods
//
//  Created by Nick Kuh on 20/04/2015.
//  Copyright (c) 2015 Mumbo Apps Ltd. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidAppear(animated: Bool) {
        super.viewDidAppear(animated)
        
        
        SVProgressHUD.showWithStatus("Hi There!!", maskType: SVProgressHUDMaskType.Gradient)
    }
    


}

